### PR TITLE
Add support for email_body and email_subject fields

### PR DESCRIPTION
**Pull request by Nightfirecat**
Closes jdm-contrib/justdelete.me#180

### CODE REVIEW

**Comment by tupaschoal** [comment]
Do note that I'm not asking for 4 PRs, but 4 commits and then we don't squash them :)

**Comment by Nightfirecat** [comment]
In general, I don't disagree--I just want every commit to self-contain any differences from re-running `gulp`. I'll see if I can split these up.

File: sites.json [hunk 1]
```diff
@@ -3616,6 +3619,8 @@
         "difficulty": "impossible",
         "notes": "Accounts may be disabled, but some data will be retained. E-mail support@riotgames.com with the subject \"Account Deactivation\" and include your username, summoner name, server and email used when registering the account in your ticket",
         "email": "support@riotgames.com",
+        "email_subject": "Account Deactivation",
+        "email_body": "Please deactivate my account%0AMyusername is XXXXXX, my summoner name is XXXXXX, I play on the XXXXXX server, and XXXXXX is my registered email address.",
```
Comment (tupaschoal) [inline sites.json]: Is the %0A there intentional?

Comment (Nightfirecat) [reply sites.json]: Yes; it is an escaped newline

**Pull request merged**